packages/contracts/contracts/interfaces/ITransactionManager.sol

```diff
@@ -11,6 +11,7 @@ interface ITransactionManager {
     address receivingChainTxManagerAddress;
     address user;
     address router;
+    address initiator; // msg.sender of sending side
```

Comment:
I probs missed a beat somewhere, but won't initiator always be user essentially?

Reply:
unless they're using a smart contract